src/components/category-form/category-form.js

```diff
@@ -0,0 +1,249 @@
+    if (e.target.files && e.target.files[0]) {
+      const reader = new FileReader();
+      reader.onload = (event) => {
+        setFieldValue('categoryImage', event.target.result);
+        setCategoryImage(event.target.result);
+      };
+      reader.readAsDataURL(e.target.files[0]);
+      setUpload(e.target.files[0]);
+    }
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <Grid item xs={12}>
+          <Paper className={styles.categoryItemUpdate}>
+            <span className={styles.imageUpload}>
+              {config.labels.avatarText}
+            </span>
+            <ImageUploadContainer
+              handler={handleImageLoad}
+              srcForAvatar={
+                categoryImage ||
+                `${config.imagePrefix}${values.categoryImage}`
+              }
+                fileName={upload.name ||category.images.thumbnail}
+            />
+            <TextField
+              data-cy='code'
+              name='code'
+              className={styles.textField}
+              variant='outlined'
+              label={'Код категорії'}
+              value={values.code}
+              onChange={handleChange}
+              error={touched.code && !!errors.code}
+            />
+            {touched.code && errors.code && (
+              <div data-cy='code-error' className={styles.inputError}>
+                {errors.code}
+              </div>
+            )}
+          </Paper>
+        </Grid>
+        <AppBar position='static'>
+          <Tabs
+            className={styles.tabs}
+            value={tabsValue}
+            onChange={handleTabsChange}
+            aria-label='simple tabs example'
+          >
+            {languageTabs}
+          </Tabs>
+        </AppBar>
+        {languages.map((lang, index) => (
+          <TabPanel key={index} value={tabsValue} index={index}>
+            <Paper className={styles.categoryItemUpdate}>
+               <TextField
+                data-cy={`${lang}Name`}
+                name={`${lang}Name`}
+                className={styles.textField}
+                variant='outlined'
+                label='Назва категорії'
+                multiline
+                value={values[`${lang}Name`]}
+                onChange={handleChange}
+                error={touched[`${lang}Name`] && !!errors[`${lang}Name`]}
+              /> 
+              {touched[`${lang}Name`] && errors[`${lang}Name`] && (
+                <div
+                  data-cy={`${lang}Name-error`}
+                  className={styles.inputError}
+                >
+                  {errors[`${lang}Name`]}
+                </div>
+              )}
+            </Paper>
+          </TabPanel>
+        ))}
+
+        <Button
+          id='contactsBack'
+          component={Link}
+          to={config.routes.pathToCategories}
+          variant='outlined'
+          color='primary'
+          className={styles.returnButton}
+          data-cy='goBackButton'
```

Comment:
Why are you using camel case here. It's better to set kebab case.

Reply:
As far as I know we use kebab case only with naming of our folders and files

Reply:
In cypress tests to pick element we are using data-cy attribute, that named by kebab case.

Reply:
![image](https://user-images.githubusercontent.com/59802190/101043323-e04d8a80-3586-11eb-83b9-41f1f977f029.png)
****
For example category cypress tests.

Reply:
Last time I was using the  "back-btn" naming
![image](https://user-images.githubusercontent.com/31392756/101057187-10e3f300-358c-11eb-9c14-f4d2bdcc3cd4.png)

Reply:
allready changed